packages/personal-note-block/src/components/NoteHeader.tsx

```diff
@@ -26,7 +26,7 @@ export const NoteHeader: FC<NoteHeaderProps> = ({
             {hasAvatarName && <span className={useLightText ? 'tw-text-white' : 'tw-text-black'}>{name}</span>}
             {hasDateEdited && dateEdited && (
                 <span className={useLightText ? 'tw-text-white tw-opacity-60' : 'tw-text-x-weak'}>
-                    Last edited on {dayjs(dateEdited).format('L, LT')}
+                    Last edited on {dayjs(dateEdited).isValid() ? dayjs(dateEdited).format('L LT') : dateEdited}
```

Comment:
Why `dayjs` and not `Intl.DateTimeFormat`? https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

Reply:
We get many different date formats from the old blocks with the migration, so dayjs is able to parse (hopefully) most of them.

Reply:
Then migrate the date during migration so we have a great baseline to start